src/mainwindow.ui

```diff
@@ -580,18 +590,44 @@
     <string>Progressive</string>
    </property>
   </action>
-  <action name="actionGPU">
+  <action name="actionNative8bitCpu">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
-    <string>GPU Effects (unstable)</string>
+    <string>Native 8-bit CPU</string>
    </property>
-   <property name="toolTip">
-    <string>Use GPU filters</string>
+  </action>
+  <action name="actionLinear8bitCpu">
+   <property name="checkable">
+    <bool>true</bool>
    </property>
-   <property name="visible">
-    <bool>false</bool>
+   <property name="text">
+    <string>Linear 8-bit CPU</string>
+   </property>
+  </action>
+  <action name="actionNative10bitCpu">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Native 10-bit CPU</string>
+   </property>
+  </action>
+  <action name="actionLinear10bitCpu">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Linear 10-bit CPU</string>
+   </property>
+  </action>
+  <action name="actionNative10bitGpuCpu">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Native 10-bit GPU/CPU (Experimental)</string>
```

Comment:
Movit always converts inputs to linear color. I have been telling people for years that GPU Effects is 10-bit linear. With your MLT changes if you supply `mlt_color_trc=linear` on the consumer, then filter `movit.convert` will keep the output from its pipeline as linear. Then, all of the CPU filters operate in linear as well. I guess the downside of doing that is I have to check that `color_transform` can properly output HLG HDR from linear as I did for Movit. If I do not get that working then we will not change `VideoWidget` below to include `mlt_color_trc=linear` for this case (currently missing). That would put this into a hybrid Linear GPU-Native CPU mode, which is fine for now. However, I still want this to say Linear because Native contradicts years of claims!